Source/Utils/WarpXUtil.cpp

```diff
@@ -446,6 +447,13 @@ void ReadBCParams ()
             }
 
         }
+        if (WarpX::field_boundary_lo[idim] == FieldBoundaryType::Damped ||
+            WarpX::field_boundary_hi[idim] == FieldBoundaryType::Damped ) {
+            int maxwell_solver_id = GetAlgorithmInteger(pp_algo, "maxwell_solver");
+            if (maxwell_solver_id != MaxwellSolverAlgo::PSATD) {
+                amrex::Abort("FieldBoundaryType::Damped is only supported for PSATD");
```

Comment:
I kept this abort here for non-PSATD simulations that have set Damped. @EZoni and @dpgrote please let me know what you think?

Reply:
This check should also be in WarpX.cpp, by the other check you added there.

Reply:
I think I agree with Dave that we could move this check to WarpX.cpp, together with the other one, so that we have both checks in one single place.